CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+Contributions are accepted as pull requests.  Please observe our coding
```

Comment:
There's a double space here. I was a graphic designer for a while and I had to lay out a book. The author had left in double spaces everywhere which I had to fix by hand. So what I am saying is that it doesn't really matter, I just don't like double spaces anymore :D